x-pack/plugins/apm/public/components/app/transaction_details/waterfall_with_summary/waterfall_container/Waterfall/waterfall_item.tsx

```diff
@@ -227,3 +235,29 @@ export function WaterfallItem({
     </Container>
   );
 }
+
+function getItemBarStyle(item: IWaterfallSpanOrTransaction, color: string, width: number, left: number): React.CSSProperties {
+  var itemBarStyle = { left: `${left}%`, width: `${width}%` };
+  
+  if(item.docType === 'span'){
+    const spanItem = item as IWaterfallSpan;
```

Comment:
Same here, I don't think you need to manually type it.

```suggestion
    const spanItem = item;
```